src/modules/components/dao/daoDataListItem/daoDataListItem.tsx

```diff
@@ -0,0 +1,45 @@
+import type React from 'react';
+import { DataList, Heading, Icon, IconType } from '../../../../core';
+import { DaoAvatar, type IDaoAvatarProps } from '../daoAvatar';
+
+export interface IDaoDataListItemProps extends IDaoAvatarProps {
+    description?: string;
+    daoAddressOrEns?: string;
+    plugin?: string;
+    network?: string;
```

Comment:
Add a short documentation for each property, the documentation will be shown on storybook